prow/plank/controller.go

```diff
@@ -375,7 +380,10 @@ func (c *Controller) syncPendingJob(pj prowapi.ProwJob, pm map[string]coreapi.Po
 			if !ok {
 				return fmt.Errorf("unknown pod %s: unknown cluster alias %q", pod.Name, pj.ClusterAlias())
 			}
-			return client.Delete(pj.ObjectMeta.Name, &metav1.DeleteOptions{})
+
+			err := client.Delete(pj.ObjectMeta.Name, &metav1.DeleteOptions{})
+			c.log.WithField("name", pj.ObjectMeta.Name).Debug("Delete Pod.")
```

Comment:
What about adding the error as well?

Reply:
I figured these were already collected so I did not log it. https://github.com/kubernetes/test-infra/blob/c6b4dedc69a097d9cd4d37292514310065e4b3ac/prow/plank/controller.go#L287

Reply:
okay. Tiny tiny nit then, move the log above the `Delete` call and keep the `return client.Delete`? Right now I intuitively expect the error to be used in the log from the way the code is structured

Reply:
Updated! That is much cleaner.